src/app/scripts/cac/pages/cac-pages-home.js

```diff
@@ -306,12 +313,12 @@ CAC.Pages.Home = (function ($, ModeOptions,  MapControl, TripOptions, SearchPara
     // Note that components are responsible for doing the right thing based on whether they're
     // active or not.
     function onUrlChanged() {
-        setActiveTab();
-        modeOptionsControl.setMode(UserPreferences.getPreference('mode'));
-        directionsControl.setFromUserPreferences();
-        exploreControl.setFromUserPreferences();
         directionsFormControl.setFromUserPreferences();
+        directionsControl.setFromUserPreferences();
+        modeOptionsControl.setMode(UserPreferences.getPreference('mode'));
         showHideNeedWheelsBanner();
+        setActiveTab();
+        exploreControl.setFromUserPreferences();
```

Comment:
There's some issues around ordering of events there. Setting the tab then fires the tab change handler, which would be the appropriate place to handle view change, but the settings may not yet be correct if `setFromUserPreferences` has not yet been called... but at least one `setFromUserPreferences` method expects the tab to have already been set.

I fiddled with several things before settling on something that worked, so not sure if this particular ordering change is actually still necessary or not.